release-calendar/src/app.ts

```diff
@@ -13,9 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import {createConnection, Connection} from 'typeorm';
-import {Release} from './entities/release';
+import 'reflect-metadata';
```

Comment:
What is this for?

Reply:
https://typeorm.io/#/
I had put it there for installing typeorm. Its also why I added` "emitDecoratorMetadata": true,` to the tsconfig.

Reply:
Hmm I used it just fine without either? Not a big deal